inc/setup.php

```diff
@@ -118,23 +118,42 @@ function uds_wp_custom_excerpt_more( $more ) {
 	}
 }
 
-add_filter( 'wp_trim_excerpt', 'uds_wp_all_excerpts_get_more_link' );
+add_filter( 'excerpt_length', 'uds_wp_custom_excerpt_length', 999 );
 
-if ( ! function_exists( 'uds_wp_all_excerpts_get_more_link' ) ) {
+if ( ! function_exists( 'uds_wp_custom_excerpt_length' ) ) {
 	/**
-	 * Adds a custom read more link to all excerpts, manually or automatically generated
+	 * Set a max number of words for excerpt.
 	 *
-	 * @param string $post_excerpt Posts's excerpt.
+	 * @param int $length the number of words in excerpt.
+	 */
+	function uds_wp_custom_excerpt_length( $length ) {
+		return 50;
```

Comment:
50 words is a bit long for our cards, that is around 300 characters. The XD design docs show News and Events cards running around 150 characters. How do you feel about trimming our excerpts to 30 words?

Reply:
Totally works for me. I also felt like the 50-word mark was too big

[4.7 characters per word](https://www.google.com/search?q=average+word+length+english&oq=average+word+length+english&aqs=chrome..69i57j0i22i30l2j0i390l2.7815j0j7&sourceid=chrome&ie=UTF-8) x 30 words = 141 characters. Seems to be in the right sweet spot.